source/xep/omemo/CToCxxProxy.cpp

```diff
@@ -68,5 +68,5 @@ void CToCxxProxyShowMessageToUser(void* proxy, char* title, char* msg)
 }
 void CToCxxProxy::showMessageToUser(char* title, char* msg)
 {
-    lurchAdapter_->showMessageToUser(title, msg);
+    //lurchAdapter_->showMessageToUser(title, msg);
```

Comment:
Not needed?

Reply:
I removed it because it can be called a lot of times when dealing with messages that cannot be decrypted.

Reply:
If a message can not be decrypted, it should be shown on the Conversation Page. To silently ignore the error, is not a solution. At least one should know that a jid tries to send a omemo encrypted message. If this is commented out completely, also other errors or warnings are not shown to the user.